app/models/invalid_identities.rb

```diff
@@ -0,0 +1,20 @@
+class InvalidIdentities
+
```

Comment:
@wtholt Yeah, I'd be much happier if this wasn't in the models directory. Semantically, I'd rather have this piece of code be a module, and not a class at all, since having it be a class implies it's an object type definition, not just a bit of code.

I'm totally on board with the "cleaner code" benefit of separating it out (to a degree), but is there a specific reason it needs to be a class in particular? Seems like this is the exact situation modules exist for in the first place.